manager/js/urls.js

```diff
@@ -65,11 +65,28 @@ function addUnitToView(unit) {
 }
 
 let prefix = '';
+let name = '';
+let label = '';
 {
   let params = (new URL(location)).searchParams;
   if (params.has('prefix')) {
     prefix = params.get('prefix');
+    $('#prefix').val(prefix);
   }
+  if (params.has('name')) {
+    name = params.get('name');
+    $('#name').val(name);
+  }
+  if (params.has('label')) {
+    label = params.get('label');
+    $('#label').val(label);
+  }
+}
+
+// prefix is required
+if (!prefix) {
```

Comment:
I would do the error handling the following way: If the prefix is empty, just show no units at all, and skip the loading of the units from the server. Show a label next to the input box with an error (popups are annoying, especially in internal tools ;) )

throwing an exception has a similar effect, but it feels like an overkill :)

Reply:
`alert` was the simple solution. This page is not supported by a UI framework. We can drop that one line, if it is too annoying... The prefix is explicitly required anyway.

Throwing the error is a perfect way to skip the rest of the script.